src/main/kotlin/mate/academy/model/Square.kt

```diff
@@ -0,0 +1,13 @@
+package mate.academy.model
+
+class Square(color: Color,

```

Comment:
Please check formatting here and in similar situations.
You may use CTRL+ALT+L hotkey for this purpose